src/bridge/EspressoTEEVerifier.sol

```diff
@@ -14,50 +14,85 @@ import {
     ENCLAVE_REPORT_LENGTH
 } from "@automata-network/dcap-attestation/contracts/types/Constants.sol";
 import {EnclaveReport} from "@automata-network/dcap-attestation/contracts/types/V3Structs.sol";
+import {
+    V3QuoteVerifier
+} from "@automata-network/dcap-attestation/contracts/verifiers/V3QuoteVerifier.sol";
+import {BytesUtils} from "@automata-network/dcap-attestation/contracts/utils/BytesUtils.sol";
+import {Ownable} from "solady/auth/Ownable.sol";
```

Comment:
Is there a particular reason we're not using the openzeppelin `Ownable` here?

Reply:
Automata uses this library for ownable. Apparently is more gas efficient